probe-rs/src/rtt/channel.rs

```diff
@@ -296,7 +300,7 @@ impl DownChannel {
     /// may not write all of `buf`.
     pub fn write(&self, core: &mut Core, mut buf: &[u8]) -> Result<usize, Error> {
         self.0.validate_core_id(core)?;
-        let (mut write, read) = self.0.read_pointers(core, "down")?;
+        let (mut write, read) = self.0.read_pointers(core, "down ")?;
```

Comment:
I'm sure it is, but just checking, is this space deliberate?

Reply:
Yes, together with:
- https://github.com/probe-rs/probe-rs/pull/2340/files#diff-88e33ffb10eef896d80e931f3ae1bd2393f62d804331d6e2f27a27631d1d5fdcR92
- and https://github.com/probe-rs/probe-rs/pull/2340/files#diff-88e33ffb10eef896d80e931f3ae1bd2393f62d804331d6e2f27a27631d1d5fdcR127

The point is to not print up or down during initialization, when we don't actually know if the channel is up or down. That being said, I'm missing the quote marks.